dataprofiler/labelers/data_labelers.py

```diff
@@ -48,6 +50,12 @@ def train_structured_labeler(data, save_dirpath=None, epochs=2):
 
     data_labeler = DataLabeler(labeler_type='structured', trainable=True)
     labels = value_label_df[1].unique().tolist()
+
+    if default_label:
```

Comment:
Don't we need to ensure that `default_label` is in `labels` before we set it as the default? I thought I remember someone mentioning this was a requirement.

Reply:
I believe this will be validated by the model on fit

Reply:
^